core/registry.js

```diff
@@ -117,7 +122,7 @@ Blockly.registry.Type.BLOCK_DRAGGER =
  *     an already registered item.
  * @throws {Error} if the type or name is empty, a name with the given type has
  *     already been registered, or if the given class or object is not valid for
- * it's type.
+ *     it's type.
```

Comment:
the nitpickiest nit ever: `its type` (i think this commit is in a different PR, sorry)